modules/rosa-additional-principals-overview.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-accessing-cluster.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.adoc
+:_mod-docs-content-type: CONCEPT
+[id="rosa-additional-principals-overview_{context}"]
+= Additional principals on your ROSA cluster
+
+You can allow AWS IAM Roles as additional principals to connect to your cluster's private API server endpoint.
+
+You can access your {hcp-title} cluster's API Server endpoint from either the public internet or the interface endpoint that was created within the VPC private subnets. By default, you can privately access your {hcp-title} API Server by using the `-kube-system-kube-controller-manager` Operator role. You must include IAM Roles from coss-account as additional allowed principals to be able to allow {hcp-title} API servce access by using another trusted AWS account. This feature offers a way to simplify your network architecture and reduced transfer costs as well as avoiding peering or attaching additional VPCs to cluster's VPC.
```

Comment:
Typo... _cross-account_

Reply:
Which one reads well
You must include IAM Roles from coss-account as additional allowed principals to be able to allow ROSA with HCP API servce access by using another trusted AWS account.

To be able to access ROSA with HCP API server from another account directly without using the primary account where cluster is installed, you must include cross-account IAM Roles as additional principals.

Reply:
Typo... HCP API _server_

Reply:
_reduced transfer costs as well as avoiding peering or attaching additional VPCs to cluster’s VPC._

This feature allows you to simplify your network architecture and reduce data transfer costs by avoiding peering or attaching cross-account VPCs to cluster's VPC.